README.md

```diff
@@ -1,6 +1,6 @@
 # Cog
 
-[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/CreatEDHack/cog)
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/prototriangle/cog)
```

Comment:
This should probably stay as CreatEDHack if this is being merged into CreatEDHack